map/local_ads_manager.hpp

```diff
@@ -82,6 +82,8 @@ class LocalAdsManager final
 
   void FillSupportedTypes();
 
+  bool DownloadCampaign(MwmSet::MwmId const & mwmId, std::vector<uint8_t> & bytes);
```

Comment:
Please, add a comment to the method - when it return true, when false, what it does with return arguments?